handbook/product/design/design-and-interaction-guidelines.md

```diff
@@ -67,19 +67,18 @@ Drawing the line between what is a code page and what is a web application page
 
 In these cases, we can utilize web application styles inside the search context to give links greater importance through color and provide additional margin and padding to aid readability.
 
+## Accessibility standards
+People of all abilities should have first-class access to code and coding. In order to achieve this it is vital that our applications are designed and built to be accessible to all.
+
+We aim for [WCAG Level AA Conformance](https://www.w3.org/WAI/WCAG2AA-Conformance), use [this reference](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa&technologies=smil%2Cflash%2Csl) to get a better understanding of what this involves.
```

Comment:
My only comment here is that "aim for" allows us to not reach this level of conformance. Maybe something like:

```suggestion
Our criteria is to meet [WCAG Level AA Conformance](https://www.w3.org/WAI/WCAG2AA-Conformance). Use [this reference](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&levels=aaa&technologies=smil%2Cflash%2Csl) to get a better understanding of what this involves.
```

Reply:
@quinnkeast curious if in any case WCAG AA conformance ever contradicted what you believed to be the best user experience for a user with accessibility needs (e.g. see  https://github.com/w3c/wcag/issues/695), what would you want to do?

"aim" allows us to do what we believe is best, which may be good unless we have a _mandate_ to meet this standard to e.g. sell our product to a government agency (afaik we don't yet).

Reply:
It's weighing this up with some areas where the standard isn't always perfect as Felix mentioned. I find your version much more significant though and I think for the vast majority of cases that's exactly what we want so will update!

Reply:
@felixfbecker I agree but I'd be curious to understand if we have any examples where we actually haven't been able to adhere to these standards (e.g. where your example applies). It seems like it is best (at least here) to be as clear as possible.

Reply:
OK, after better understanding the implications of that kind of issue I'm flip-flopping on this. 

How about:
> We aim for WCAG Level AA Conformance in all areas. Use this reference to get a better understanding of what this involves.

Worded slightly differently for significance. I think ultimately we understand what we need to achieve the actual wording here should have minimal difference unless we are legally required to declare this

Reply:
I'm also flip-flopping, because of @felixfbecker's example, which I strongly agree is a big problem with the guidelines using the guidelines a rote reference. The missing link for me is in how we convey "accessibility is not optional and isn't scope that can be cut in future work."